inst/tests/tests.Rraw

```diff
@@ -7639,17 +7639,19 @@ test(1588.1, dt[f %in% 3:4], dt[3:4])
 test(1588.2, dt[f == 3], dt[3])
 
 # encoding issue in forder
-x <- "fa\xE7ile"
-Encoding(x)
-Encoding(x) <- "latin1"
-xx <- iconv(x, "latin1", "UTF-8")
-y = sample(c(x,xx), 10, TRUE)
-oy = if (length(oy <- forderv(y))) oy else seq_along(y)
-test(1590.4, oy, order(y))
-Encoding(xx) = "unknown"
-y = sample(c(x,xx), 10, TRUE)
-oy = if (length(oy <- forderv(y))) oy else seq_along(y)
-test(1590.5, oy, order(y))
+if (!memtest) {
```

Comment:
any idea why calling `gc()` and `system()` in each `test()` makes this test failing? @mattdowle

Reply:
Not sure either. What's the error message?  Maybe there's a true memory fault here that the additional `gc()` tickles out.  One option would be to run `test.data.table()` with memtest on in R-devel compiled with `--strict-barrier` and `ASAN`, as per CRAN_Release.cmd commands.  That has a good chance of giving us a line number at the root cause.  If not, then it's valgrind in an instrumented build of R, possibly with gctorture on too (see commands in CRAN_Release.cmd).
Suggest raising a new issue for that though, to look at separately. No need to hold up this PR because of this.